api-reference/media-api/list-and-search-files.md

```diff
@@ -1114,4 +1172,20 @@ resp, err := ik.Media.Files(ctx, media.FilesParam{
 })
 ```
 {% endtab %}
+
+{% tab title=".Net" %}
+```.net
+var imagekit = new ImageKit({
+    publicKey : "your_public_api_key",
+    privateKey : "your_private_api_key",
+    urlEndpoint : "https://ik.imagekit.io/your_imagekit_id/"
+});
+
+GetFileListRequest model = new GetFileListRequest
+    { 
+        SearchQuery = 'format="png"'
```

Comment:
At some point you mentioned that in .NET single quote won't work like this so you are doing "format=\"png\"".

Reply:
It will work if we are using direct string instead of string builder class